Sources/HipMobileUI/Pages/ExhibitDetailsPage.xaml.cs

```diff
@@ -101,14 +91,7 @@ private void ViewModelOnPropertyChanged(object sender, PropertyChangedEventArgs
         /// </summary>
         private void ToggleAudioBarVisibility()
         {
-            if (ViewModel.AudioToolbarVisible)
-            {
-                AudioToolbar.TranslateTo(0, 0);
-            }
-            else
-            {
-                AudioToolbar.TranslateTo(0, -100);
-            }
+                AudioToolbar.IsVisible= ViewModel.AudioToolbarVisible;
```

Comment:
Nitpick: There should be a space before the `=`. Also the indentation looks wrong? Try STRG+K STRG+D.